package.json

```diff
@@ -77,6 +77,8 @@
     "@ag-grid-community/client-side-row-model": "31.3.2",
     "@ag-grid-community/react": "^31.3.2",
     "@ag-grid-community/styles": "^31.3.2",
+    "@faker-js/faker": "8.3.1",
+    "@graasp/sdk": "4.14.1",
```

Comment:
If you need these when developing, you should add them to the `devDependencies` I am not sure they need to be direct dependencies of the package.